Index: src/main/java/org/example/DAOs/user.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package org.example.DAOs;\r\n\r\npublic class user {\r\n}\r\n
===================================================================
diff --git a/src/main/java/org/example/DAOs/user.java b/src/main/java/org/example/DAOs/user.java
--- a/src/main/java/org/example/DAOs/user.java	
+++ b/src/main/java/org/example/DAOs/user.java	
@@ -1,4 +1,9 @@
 package org.example.DAOs;
 
-public class user {
+import org.example.entities.User;
+
+public class user extends GenericDAOImpl<User, Integer> {
+    public user() {
+        super(User.class);
+    }
 }
