Index: src/main/java/org/example/DAOs/UserDAO.java
===================================================================
diff --git a/src/main/java/org/example/DAOs/UserDAO.java b/src/main/java/org/example/DAOs/UserDAO.java
deleted file mode 100644
--- a/src/main/java/org/example/DAOs/UserDAO.java	(revision 4c037b1285d8c77b34d066861aa51618642460e4)
+++ /dev/null	(revision 4c037b1285d8c77b34d066861aa51618642460e4)
@@ -1,41 +0,0 @@
-package org.example.DAOs;
-
-import org.example.entities.*;
-
-import java.util.List;
-
-public class UserDAO extends GenericDAOImpl<User, Integer> {
-    public UserDAO() {
-        super(User.class);
-    }
-    public static void main(String[] args) {
-        // Create an instance of UserDAOImpl
-        UserDAO userDAO = new UserDAO();
-
-        // Create a new User instance
-        User newUser = new User();
-        newUser.setId(3);
-        newUser.setName("John Doe");
-        newUser.setEmail("john3.doe@example.com");
-        newUser.setPassword("password123");
-        newUser.setJob("Developer");
-        newUser.setCredit("5000");
-        newUser.setCity("New York");
-        newUser.setStreet("5th Avenue");
-
-        // Save the User to the database
-        userDAO.save(newUser);
-        System.out.println("User created successfully!");
-
-        // Retrieve all Users from the database
-        List<User> allUsers = userDAO.findAll();
-
-        // Print all User details
-        System.out.println("All Users:");
-        for (User user : allUsers) {
-            System.out.println(user);
-        }
-
-
-    }
-}
\ No newline at end of file
Index: src/main/java/org/example/DAOs/CartItemDAO.java
===================================================================
diff --git a/src/main/java/org/example/DAOs/CartItemDAO.java b/src/main/java/org/example/DAOs/CartItemDAO.java
deleted file mode 100644
--- a/src/main/java/org/example/DAOs/CartItemDAO.java	(revision 4c037b1285d8c77b34d066861aa51618642460e4)
+++ /dev/null	(revision 4c037b1285d8c77b34d066861aa51618642460e4)
@@ -1,11 +0,0 @@
-package org.example.DAOs;
-
-import org.example.entities.Admin;
-import org.example.entities.Cartitem;
-
-public class CartItemDAO extends GenericDAOImpl<Cartitem, Integer> {
-    public CartItemDAO(){
-        super(Cartitem.class);
-    }
-
-}
Index: src/main/java/org/example/DAOs/AdminDAO.java
===================================================================
diff --git a/src/main/java/org/example/DAOs/AdminDAO.java b/src/main/java/org/example/DAOs/AdminDAO.java
deleted file mode 100644
--- a/src/main/java/org/example/DAOs/AdminDAO.java	(revision 4c037b1285d8c77b34d066861aa51618642460e4)
+++ /dev/null	(revision 4c037b1285d8c77b34d066861aa51618642460e4)
@@ -1,11 +0,0 @@
-package org.example.DAOs;
-
-import org.example.entities.Admin;
-import org.example.entities.User;
-
-public class AdminDAO extends GenericDAOImpl<Admin, Integer> {
-    public AdminDAO(){
-        super(Admin.class);
-    }
-
-}
Index: src/main/java/org/example/DAOs/WishlistDAO.java
===================================================================
diff --git a/src/main/java/org/example/DAOs/WishlistDAO.java b/src/main/java/org/example/DAOs/WishlistDAO.java
deleted file mode 100644
--- a/src/main/java/org/example/DAOs/WishlistDAO.java	(revision 4c037b1285d8c77b34d066861aa51618642460e4)
+++ /dev/null	(revision 4c037b1285d8c77b34d066861aa51618642460e4)
@@ -1,13 +0,0 @@
-package org.example.DAOs;
-
-import org.example.entities.Userinterest;
-import org.example.entities.Wishlist;  // Make sure the Wishlist entity is properly imported
-import jakarta.persistence.EntityManager;
-import jakarta.persistence.EntityManagerFactory;
-import org.example.entities.WishlistId;
-
-public class WishlistDAO extends GenericDAOImpl<Wishlist, Integer> {
-    public WishlistDAO() {
-        super(Wishlist.class);
-    }
-}
Index: src/main/java/org/example/DAOs/CategoryDAO.java
===================================================================
diff --git a/src/main/java/org/example/DAOs/CategoryDAO.java b/src/main/java/org/example/DAOs/CategoryDAO.java
deleted file mode 100644
--- a/src/main/java/org/example/DAOs/CategoryDAO.java	(revision 4c037b1285d8c77b34d066861aa51618642460e4)
+++ /dev/null	(revision 4c037b1285d8c77b34d066861aa51618642460e4)
@@ -1,12 +0,0 @@
-package org.example.DAOs;
-
-import org.example.entities.Cartitem;
-import org.example.entities.Category;  // Make sure the Category entity is properly imported
-import jakarta.persistence.EntityManager;
-import jakarta.persistence.EntityManagerFactory;
-
-public class CategoryDAO extends GenericDAOImpl<Category, Integer> {
-    public CategoryDAO(){
-        super(Category.class);
-    }
-}
Index: src/main/java/org/example/DAOs/OrderDAO.java
===================================================================
diff --git a/src/main/java/org/example/DAOs/OrderDAO.java b/src/main/java/org/example/DAOs/OrderDAO.java
deleted file mode 100644
--- a/src/main/java/org/example/DAOs/OrderDAO.java	(revision 4c037b1285d8c77b34d066861aa51618642460e4)
+++ /dev/null	(revision 4c037b1285d8c77b34d066861aa51618642460e4)
@@ -1,12 +0,0 @@
-package org.example.DAOs;
-
-import org.example.entities.Category;
-import org.example.entities.Order;  // Make sure the Order entity is properly imported
-import jakarta.persistence.EntityManager;
-import jakarta.persistence.EntityManagerFactory;
-
-public class OrderDAO extends GenericDAOImpl<Order, Integer> {
-    public OrderDAO(){
-        super(Order.class);
-    }
-}
Index: src/main/java/org/example/DAOs/OrderItemDAO.java
===================================================================
diff --git a/src/main/java/org/example/DAOs/OrderItemDAO.java b/src/main/java/org/example/DAOs/OrderItemDAO.java
deleted file mode 100644
--- a/src/main/java/org/example/DAOs/OrderItemDAO.java	(revision 4c037b1285d8c77b34d066861aa51618642460e4)
+++ /dev/null	(revision 4c037b1285d8c77b34d066861aa51618642460e4)
@@ -1,13 +0,0 @@
-package org.example.DAOs;
-
-import org.example.entities.Order;
-import org.example.entities.Orderitem;
-import org.example.entities.OrderitemId; // Make sure the OrderItem entity is properly imported
-import jakarta.persistence.EntityManager;
-import jakarta.persistence.EntityManagerFactory;
-
-public class OrderItemDAO extends GenericDAOImpl<Orderitem, Integer> {
-    public OrderItemDAO(){
-        super(Orderitem.class);
-    }
-}
\ No newline at end of file
Index: src/main/java/org/example/DAOs/ProductDAO.java
===================================================================
diff --git a/src/main/java/org/example/DAOs/ProductDAO.java b/src/main/java/org/example/DAOs/ProductDAO.java
deleted file mode 100644
--- a/src/main/java/org/example/DAOs/ProductDAO.java	(revision 4c037b1285d8c77b34d066861aa51618642460e4)
+++ /dev/null	(revision 4c037b1285d8c77b34d066861aa51618642460e4)
@@ -1,12 +0,0 @@
-package org.example.DAOs;
-
-import org.example.entities.Orderitem;
-import org.example.entities.Product;  // Make sure the Product entity is properly imported
-import jakarta.persistence.EntityManager;
-import jakarta.persistence.EntityManagerFactory;
-
-public class ProductDAO extends GenericDAOImpl<Product, Integer> {
-    public ProductDAO(){
-        super(Product.class);
-    }
-}
Index: src/main/java/org/example/DAOs/UserInterestDAO.java
===================================================================
diff --git a/src/main/java/org/example/DAOs/UserInterestDAO.java b/src/main/java/org/example/DAOs/UserInterestDAO.java
deleted file mode 100644
--- a/src/main/java/org/example/DAOs/UserInterestDAO.java	(revision 4c037b1285d8c77b34d066861aa51618642460e4)
+++ /dev/null	(revision 4c037b1285d8c77b34d066861aa51618642460e4)
@@ -1,13 +0,0 @@
-package org.example.DAOs;
-
-import org.example.entities.Userinterest;
-import org.example.entities.Userinterest;  // Make sure the UserInterest entity is properly imported
-import jakarta.persistence.EntityManager;
-import jakarta.persistence.EntityManagerFactory;
-import org.example.entities.UserinterestId;
-
-public class UserInterestDAO extends GenericDAOImpl<Userinterest, Integer> {
-    public UserInterestDAO() {
-        super(Userinterest.class);
-    }
-}
Index: src/main/java/org/example/DAOs/EntityManagerFactorySinglton.java
===================================================================
diff --git a/src/main/java/org/example/DAOs/EntityManagerFactorySinglton.java b/src/main/java/org/example/DAOs/EntityManagerFactorySinglton.java
deleted file mode 100644
--- a/src/main/java/org/example/DAOs/EntityManagerFactorySinglton.java	(revision 4c037b1285d8c77b34d066861aa51618642460e4)
+++ /dev/null	(revision 4c037b1285d8c77b34d066861aa51618642460e4)
@@ -1,15 +0,0 @@
-package org.example.DAOs;
-
-import jakarta.persistence.EntityManagerFactory;
-import jakarta.persistence.Persistence;
-
-public class EntityManagerFactorySinglton {
-    private static EntityManagerFactory emf;
-
-    public synchronized EntityManagerFactory getEntityManagerFactory(){
-        if(emf == null){
-            return emf = Persistence.createEntityManagerFactory("noura");
-        }
-        return emf;
-    }
-}
